src/Env/Config/Config.php

```diff
@@ -42,28 +50,25 @@ public function getFiles()
     {
         $origin = $this->getOrigin();
 
-        yield ($origin instanceof \SplFileInfo) ? $origin : new \SplFileInfo($origin);
+        yield ($origin instanceof SplFileInfo) ? $origin : new SplFileInfo($origin);
```

Comment:
Simply because it is used many times in the class but I'll remove the use, after all there should not have any gain in doing that.